.github/workflows/beam_PostCommit_Java_Tpcds_Spark.yml

```diff
@@ -85,6 +84,7 @@ jobs:
       matrix:
         job_name: ["beam_PostCommit_Java_Tpcds_Spark"]
         job_phrase: ["Run Spark Runner Tpcds Tests"]
+        runner: [SparkRunner, SparkStructuredStreamingRunner]
```

Comment:
We should add this to both the job `name` on line 82 and the `github_job` value on line 95 (which helps configure trigger phrases)